providers/universal-provider/src/UniversalProvider.ts

```diff
@@ -231,7 +232,13 @@ export class UniversalProvider implements IUniversalProvider {
       // ignore without active session
       if (!this.session) return;
       const [namespace, chainId] = this.validateChain(chain);
-      this.getProvider(namespace).setDefaultChain(chainId, rpcUrl);
+      const provider = this.getProvider(namespace);
+      // @ts-expect-error
```

Comment:
Why do we need an `@ts-expect-error` here ?

Reply:
we're accesing private prop. I can make it public so ts doesn't complain